src/lib/views/account-settings/DeleteAccount.svelte

```diff
@@ -7,28 +7,98 @@
   import type { AppStore } from "$lib/stores/types";
   import Button from "../../../lib/components/Button.svelte";
   import isMounted from "../../../lib/is-mounted";
+
   const dispatch = createEventDispatcher();
   const store: AppStore = getContext("rally:store");
+  const errorClass = "mzp-c-field-control mzp-c-field-control--error";
+  const inputClass = "mzp-c-field-control";
+
+  let btnDisabled = true;
+  let inputPasswordClass;
+  let inputItemsVisible = false;
+  let passwordEl;
+  let errText = null;
+  let passwordVisible = false;
+  let fireBaseErr = null;
 
   const offboardURL = "https://rally.mozilla.org/account-deleted";
   const mounted = isMounted();
 
   let leaveModal = false;
+  let isGoogleAccount;
   let Dialog;
 
   onMount(async () => {
+    inputPasswordClass = inputClass;
+    const userProvider = await store.getUserProvider();
+    isGoogleAccount =
+      userProvider &&
+      userProvider.length &&
+      userProvider[0].providerId &&
```

Comment:
It's possible to [link multiple providers to a single account](https://firebase.google.com/docs/auth/web/account-linking) but we don't currently do this. So it will always be the 0th provider.

I'm not sure how one would reset the password on a Google based account, since we hide that card in Account Settings.

Reply:
If you sign out and use the forgot password link on an email you registered using the Google account firebase will attach the email provider.

Reply:
disambiguated this in latest commit